docs/configuring.rst

```diff
@@ -49,6 +50,8 @@ Base options
 * ``vcs_pageview_mode`` String. Changes how to view files when using `display_github`, `display_gitlab`, etc.
   When using Github or Gitlab this can be: `blob` (default), `edit`, or `raw`,
   on Bitbucket, this can be either: `view` (default) or `edit`.
+  * ``versioning`` Bool. Add 'versions' menu to the bottom of the sidebar. Defaults to ``False``.
```

Comment:
Could you add some documentation (either here or in a a separate section) which information is expected to be made available by the build environment? Perhaps link to some RTD code on GitHub that sets this.
This is an odd setting where just enabling it will not make it work, it depends on other information being available.

Reply:
Please, note that this is also indirectly related to `display_github`, because `github_version` should match `current_version`, as long as the version is named after the corresponding branch. Otherwise, the edit/view button will always point to the master branch, no matter which version is actually being browsed.